sklearn/manifold/tests/test_t_sne.py

```diff
@@ -1198,16 +1226,17 @@ def test_tsne_n_jobs(method):
     assert_allclose(X_tr_ref, X_tr)
 
 
-@pytest.mark.filterwarnings("ignore:The PCA initialization in TSNE will change")
 # FIXME: remove in 1.3 after deprecation of `square_distances`
-def test_tsne_deprecation_square_distances():
+@pytest.mark.filterwarnings("ignore:The PCA initialization in TSNE will change")
+@pytest.mark.parametrize("dtype", DTYPES)
+def test_tsne_deprecation_square_distances(dtype):
```

Comment:
The purpose of this test is just to check the deprecation warning. I don't think it's worth spending time and parametrization complexity on the dtypes for this kind of tests.